langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/TypeParamTest.java

```diff
@@ -50,7 +50,8 @@ public void testTypeParamNegative() {
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'boolean', found 'string'", 48, 18);
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'Foo', found 'string'", 50, 14);
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'Bar', found 'string'", 51, 14);
-        BAssertUtil.validateError(result, err++, "incompatible types: expected 'boolean', found 'BarDetail'", 65, 18);
+        BAssertUtil.validateError(result, err++, "incompatible types: expected 'boolean', found " +
```

Comment:
The bal files error comment also needs to be updated.

https://github.com/ballerina-platform/ballerina-lang/blob/88de76ededb5b0ac6eacfb795fa4e8511d9d0bf8/langlib/langlib-test/src/test/resources/test-src/type-param/type_param_test_negative.bal#L65

Reply:
Will address with this issue separately: https://github.com/ballerina-platform/ballerina-lang/issues/33598